wordpress-mobile/WordPress-iOS
# Issues/7272 split self hosted flow part 2

Refs #7272
This PR is part two of splitting the self-hosted login screens and updating to the new login flow style.  It builds upon the work in #7378 and should be reviewed after #7378 is merged.

To test:
- Test the self-hosted login flow and confirm you can login with self-hosted or wpcom sites.
- Confirm the visual design was correctly implemented.
- Confirm the form functions properly.
- Confirm the header shows the correct information for wpcom/jetpack and for self-hosted sites.  To test self-hosted sites you can just comment out line 189 of LoginSiteAddressViewController.

Needs review: @nheagy 

Examples:
![simulator screen shot jun 21 2017 4 36 26 pm](https://user-images.githubusercontent.com/1435271/27408164-f61f1bc6-56a0-11e7-9b09-d100825ef221.png)
![simulator screen shot jun 21 2017 4 41 46 pm](https://user-images.githubusercontent.com/1435271/27408167-f99a9960-56a0-11e7-99c5-17aa5b66685f.png)
